Makefile

```diff
@@ -151,7 +151,7 @@ DOCKER_BUF := $(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace $(
 ###############################################################################
 ###                                Protobuf                                 ###
 ###############################################################################
-PROTO_BUILDER_IMAGE=tendermintdev/sdk-proto-gen:v0.2
+PROTO_BUILDER_IMAGE=ghcr.io/cosmos/proto-builder:0.9.0
```

Comment:
Question is there is a list of compatible container for proto-builder? I still struggle to find how this container is built!

Reply:
here you go:

github.com/faddat/proto-builder